src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write code here
+const tableHead = document.querySelector('thead');
+const tableBody = document.querySelector('tbody');
+
+function convertSalary(salary) {
+  const noCurrency = salary.slice(1);
+
+  return +noCurrency.split(',').join('');
+}
+
+tableHead.addEventListener('click', e => {
+  const columnIndex = e.target.cellIndex;
+
+  const sortColumn = [...tableBody.children].sort((a, b) => {
+    const colA = a.querySelectorAll('td')[columnIndex].innerText;
+    const colB = b.querySelectorAll('td')[columnIndex].innerText;
+
+    if (columnIndex === 3) {
```

Comment:
Avoid using magic numbers in your code. It's better to create a constant variable to store the index of the salary column.